vscode-extension/src/tools/createPromptTemplates.ts

```diff
@@ -0,0 +1,240 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Parameters for the paw_create_prompt_templates language model tool.
+ */
+interface CreatePromptTemplatesParams {
+  /** The normalized feature slug (e.g., "auth-system") */
+  feature_slug: string;
+  
+  /** Absolute path to the workspace root directory */
+  workspace_path: string;
+}
+
+/**
+ * Result returned by the createPromptTemplates function.
+ */
+interface CreatePromptTemplatesResult {
+  /** Whether all template files were created successfully */
+  success: boolean;
+  
+  /** List of absolute paths to successfully created files */
+  files_created: string[];
+  
+  /** List of error messages encountered during creation */
+  errors: string[];
+}
+
+/**
+ * Template definitions for all 9 PAW prompt files.
+ * 
+ * Each template includes:
+ * - filename: The exact filename to use
+ * - mode: The chatmode to invoke (corresponds to .github/chatmodes/*.chatmode.md)
+ * - instruction: The action the agent should perform
+ */
+const PROMPT_TEMPLATES = [
+  {
+    filename: '01A-spec.prompt.md',
+    mode: 'PAW-01A Spec Agent',
+    instruction: 'Create spec from'
+  },
+  {
+    filename: '01B-spec-research.prompt.md',
+    mode: 'PAW-01B Spec Research Agent',
+    instruction: 'Answer research questions from'
+  },
+  {
+    filename: '02A-code-research.prompt.md',
+    mode: 'PAW-02A Code Researcher',
+    instruction: 'Run code research from'
+  },
+  {
+    filename: '02B-impl-plan.prompt.md',
+    mode: 'PAW-02B Impl Planner',
+    instruction: 'Create implementation plan from'
+  },
+  {
+    filename: '03A-implement.prompt.md',
+    mode: 'PAW-03A Implementer',
+    instruction: 'Implement phase from'
+  },
+  {
+    filename: '03B-review.prompt.md',
+    mode: 'PAW-03B Impl Reviewer',
+    instruction: 'Review implementation from'
+  },
+  {
+    filename: '04-docs.prompt.md',
+    mode: 'PAW-04 Documenter Agent',
+    instruction: 'Generate documentation from'
+  },
+  {
+    filename: '05-pr.prompt.md',
+    mode: 'PAW-05 PR Agent',
+    instruction: 'Create final PR from'
+  },
+  {
+    filename: '0X-status.prompt.md',
+    mode: 'PAW-0X Status Agent',
+    instruction: 'Update status from'
+  }
+];
```

Comment:
Add two more prompt files. These 03C-address-pr-comments and 03D-review-address-pr-comments . These prompts use the Implementer and Impl Reviewer agent, respectively. These are used when there's a PR that has comments that need to be addressed.